x/oracle/types/asset.go

```diff
@@ -7,6 +7,8 @@ import (
 // Denomination constants
 const (
 	UmeeDenom       string = "uumee"
+	UmeeSymbol      string = "umee"
```

Comment:
we technically don't need these. We store all of them in `app.go`, so we should just reference them here too 👍

Reply:
It's done this way to avoid import cycles - is there a way to get around that?

Reply:
ahhh you're totally right. Unfortunately, not. Good point